2024-12-08 23:40:35.412 -03:00 [ERR] Exception caught
System.NullReferenceException: Object reference not set to an instance of an object.
   at GestorCV.API.Controllers.Servicios.Empleos.PeticionAgregar.Procesar() in C:\Cristian\_Casa\TFI\TdC-TPFinal\servidor\GestorCV.API\Controllers\Servicios\Empleos\PeticionAgregar.cs:line 31
   at GestorCV.API.Controllers.Base.PeticionTransaccionableBase.ProcesarExtendido() in C:\Cristian\_Casa\TFI\TdC-TPFinal\servidor\GestorCV.API\Controllers\Base\PeticionTransaccionable.cs:line 17
   at GestorCV.API.Controllers.Servicios.EjecutorPeticiones.Ejecutar(IPeticion peticion) in C:\Cristian\_Casa\TFI\TdC-TPFinal\servidor\GestorCV.API\Controllers\Servicios\EjecutorPeticiones.cs:line 22
   at GestorCV.API.Controllers.EmpleosController.Agregar(Parametros parametros) in C:\Cristian\_Casa\TFI\TdC-TPFinal\servidor\GestorCV.API\Controllers\EmpleosController.cs:line 72
   at lambda_method655(Closure, Object, Object[])
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at GestorCV.API.Infraestructura.Seguridad.AutorizacionMiddleware.Invoke(HttpContext context) in C:\Cristian\_Casa\TFI\TdC-TPFinal\servidor\GestorCV.API\Infraestructura\Seguridad\AutorizacionMiddleware.cs:line 61
   at GestorCV.API.Infraestructura.ErrorHandlerMiddleware.Invoke(HttpContext context) in C:\Cristian\_Casa\TFI\TdC-TPFinal\servidor\GestorCV.API\Infraestructura\ExceptionMiddleware.cs:line 22
